api/scanner/scanner_tasks/processing_tasks/process_photo_task.go

```diff
@@ -96,11 +96,9 @@ func (t ProcessPhotoTask) ProcessMedia(ctx scanner_task.TaskContext, mediaData *
 	if origURL == nil {
```

Comment:
It's better to move the definition of `photoDimensions` in line 58 just before this `if` condition.

Reply:
Done